Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Start from the latest golang base image
+FROM golang:1.13 as builder
+
+# Set the Current Working Directory inside the container
+WORKDIR /app
+
+# Copy go mod and sum files
+COPY go.mod go.sum ./
+
+# Download all dependencies. Dependencies will be cached if the go.mod and go.sum files are not changed
+RUN go mod download
+
+# Copy the source from the current directory to the Working Directory inside the container
+COPY . .
+
+# Build the Go app
+RUN CGO_ENABLED=0 go build -o tyk .
+
+######## Start a new stage from scratch #######
+FROM debian:buster-slim
+
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get install -y --no-install-recommends ca-certificates \
```

Comment:
Much more stuff is needed to be able to use Python plugins. As I'm not sure which flows would this image support I'm not sure which things this needs.

Please take a look at this for reference:
https://github.com/TykTechnologies/tyk-gateway-docker/blob/bdd642e0261402565cd2f0f88ba1ddac7e9cbd25/Dockerfile#L9